include/rawpheno.download.form.inc

```diff
@@ -24,16 +24,29 @@ function rawpheno_download($form, &$form_state) {
     $project_selected = $form_state['values']['sel_project'];
   }
 
-  $sql = "SELECT DISTINCT t1.project_id, t1.name
-          FROM {project} AS t1 RIGHT JOIN pheno_plant_project AS t2 USING(project_id)
-          ORDER BY t1.project_id DESC";
+  // Sort the project names according to Planting Date.
+  // Put project with recently uploaded data/ based on planting year
+  // first in the list.
+  $sql = "SELECT project_id, name
+          FROM {project} AS t1
+            RIGHT JOIN pheno_plant_project AS t2 USING (project_id)
+            LEFT JOIN pheno_measurements AS t3 USING (plant_id)
+          WHERE t3.type_id = (SELECT cvterm_id FROM {cvterm} WHERE name = 'Planting Date (date)' LIMIT 1)

```

Comment:
Never use `LIMIT 1` in queries. Instead you should be looking at the unique keys and making these queries specific enough that you don't need to worry about it returning more then one result. In this specific case, you should be restricting it by cv.